ArtOfIllusion/src/artofillusion/MaterialMappingDialog.java

```diff
@@ -1,4 +1,5 @@
 /* Copyright (C) 2000-2004 by Peter Eastman
+   Changes copyright (C) 2017 by Maksim Khramov
```

Comment:
Please do this in the commit where you actually edited the file. If you forget, `git commit --amend` and `git rebase -i` are your friends. In this case, please leave things as they are until we can complete reviewing your changes. If we decide to merge the PR, I'll clean up the commit history on my end.